examples/deno.js

```diff
@@ -1,4 +1,4 @@
-import { ExtismPlugin, ExtismPluginOptions } from '../src/deno/mod.ts'
+import { createPlugin, ExtismPluginOptions } from '../src/deno/mod.ts'
 
 async function main() {
```

Comment:
You don’t need this main function for the Deno example. You can have the top-level awaits directly. Also, what about renaming this file to deno.ts?

Reply:
thanks!